live-ingester/src/integration-test/resources/expected-hxinsight-requests/delete-document-request.yml

```diff
@@ -0,0 +1,14 @@
+url: /v1/ingestion-events
```

Comment:
For this method of testing to be valid we need to ensure that we're also checking the application can output these requests. We are doing this for `create-document-request.yml` in `CreateRequestIntegrationTest.java`, but I don't think we're doing it for the other three yml files.

Reply:
I've updated testUpdateRequest() in UpdateRequestIntegrationTest.java class with "/expected-hxinsight-requests/update-document-request.yml". In UpdateRequestIntegrationTest.java there are test cases which use different request body - we do not use those in OpenApi tests. DeleteRequestIntegrationTest.java class was updated with "/expected-hxinsight-requests/delete-document-request.yml". There is no body for PresignedUrls case so no need for update there.